Core/GDCore/Extensions/Builtin/AudioExtension.cpp

```diff
@@ -49,7 +52,7 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsAudioExtension(
                  _("Stop the sound of a channel"),
                  _("Stop the sound on the specified channel."),
                  _("Stop the sound of channel _PARAM1_"),
-                 _("Audio/Sounds on channels"),
+                 _("Sounds on channels"),
                  "res/actions/son24.png",
```

Comment:
like here

Reply:
Just no time to change everything, that's the reason ;)